pkgs/by-name/zm/zmkBATx/package.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, qt6
+, pkg-config
+, dbus
+, simpleBluez
+}:
+let
+  version = "1.0.1";
+  pname = "zmkBATx";
+  src = fetchFromGitHub {
+    owner = "mh4x0f";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

Reply:
Never thought about this. Notice that now instead `src` depends on  `finalAttrs.version` but this shouldn't be a problem if I correctly understood what that link.